core/indexer/factory/indexerFactory.go

```diff
@@ -14,6 +15,11 @@ import (
 	"github.com/elastic/go-elasticsearch/v7"
 )
 
+const (
+	withKibanaFolder = "withKibana"
```

Comment:
withKibanaDir or withKibanaDirectory,  "Folder" is something related to windows only :)